pex/pex_builder.py

```diff
@@ -229,6 +229,35 @@ def add_interpreter_constraint(self, ic):
     self._ensure_unfrozen('Adding an interpreter constraint')
     self._pex_info.add_interpreter_constraint(ic)
 
+  def add_from_requirements_pex(self, pex):
+    """Add requirements from an existing pex.
+
+    :param pex: The path to an existing .pex file or unzipped pex directory.
+    """
+    self._ensure_unfrozen('Adding from pex')
+    pex_info = PexInfo.from_pex(pex)
+
+    def add(location, dname, expected_dhash):
```

Comment:
Nit: it took me a minute to figure out whether `dname` was `dependency_name` or `distribution_name` or something else. Spelling it out as `distribution_name` will save future readers this time.